src/pages/api/contact.js

```diff
@@ -14,9 +14,9 @@ export default async function handler(req, res) {
   }
 
   const text = [
-    `Name: \`${name}\``,
+    `Name: \`${name || 'Name Not Provided'}\``,
```

Comment:
@Cameron-Porter should we have a more descriptive message for when this is sent to slack identifying that it was sent to from the footer? When this is posted over it might look like a field was missed on the orinial form.

Reply:
Think something like "Field not accessible on Footer Form" and include that with the phone as well?

Reply:
Sure or sent from footer form. I'm open.

Reply:
I like that better, will have it pushed in a moment.